compositions/upbound-aws-provider/dynamo-irsa/definition.yaml

```diff
@@ -0,0 +1,163 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: apiextensions.crossplane.io/v1
+kind: CompositeResourceDefinition
+metadata:
```

Comment:
Let's add labels.

Reply:
What labels and why? The rest of the XRDs don't have labels.

Reply:
my bad I was looking at the wrong file